File: cs50/x-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int H ;

    do
    {
        H = get_int("Enter Height: ");
    }
    while(H < 1 || H > 8);

    int i;
    int l;
    int space;
    for(i = 1 ; i <= H; i++)
    {
       

       

        for( space = H - i; space > 0 ; space--)
        {
            
            printf(" ");
        }
        for(l = i; l > 0 ; l--)
        {
            printf("#");
        }
        printf("  ");
         for(l = i; l > 0 ; l--)
        {
            printf("#");
        }
        printf("\n");

    }

}

File: cs50/x-sentimental-mario-more/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height > 8 or height < 1 :
    height = int(input("Height: "))


for i in range(height):
    print(" " * (height - (i + 1)), end = '')
    print("#" * (i + 1), end = "  ")
    print("#" * (i + 1)) 
    